src/web/editor.template.html

```diff
@@ -104,7 +104,6 @@
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
 <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js"></script>
 <script src="/js/loader.js"></script>
-<script src="/js/url.js"></script>
```

Comment:
Yes. For that reason, it would be helpful to know what the implicit dependency tree is for these script tags. I'm slightly terrified to remove something here that happens to be used somewhere in the bowels of the pyret runtime, resulting in breaking everything.

The goal is to remove all globals and make all the module dependencies explicit.